working/macros/api/code.dart

```diff
@@ -1,51 +1,36 @@
-/// A scope in which to resolve a chunk of code.
-///
-/// TODO: Specify more what these mean, what fields are available (if any), etc.
-abstract class Scope {}
```

Comment:
I do think we'll probably still want this as a way to create a new identifier that explicitly resolves in a given scope. That way you can create code that refers to declarations that are themselves produced by a macro.

The old doc I wrote about the Code API sketches it out a little. But you can drop it for now if you want.

Removing the Scope parameters to all of the other APIs LGTM.

Reply:
I think removing it for now is good until we have a better understanding of how it will be used, especially with the context of augmentation libraries.